app/javascript/components/applications/application_form/HouseholdMembersSection.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react'
+import _ from 'lodash'
```

Comment:
same thing here about lodash..not sure if you worked a lot in this components and refactoring this is out of the scope of this PR

Reply:
yeah what this import is and the one above it is me adding imports where lodash was used but never imported, did so to pass overcommit, which was checking the whole codebase, not just the files i was editing. so even though this story didn't involve these components i added those imports. that was before grant pointed out that we should just import the specific lodash functions we need. i can definitely go back and update these to do that, since i'm bothering to add the imports i'll update them to use our new convention